load.go

```diff
@@ -15,6 +15,7 @@ import (
 
 // Load returns a map of Mocks
 func (om *OpenMock) Load() error {
+	logrus.Infof("Start to load templates from: %s", om.TemplatesDir)
```

Comment:
Let's move this info log to `loadYAML` function

Reply:
Done